docs/windows.md

```diff
@@ -1,142 +1,94 @@
 
-## Manual Way: download and install dependencies
+You can permanently append the Cygwin bin to your `PATH` so that on subsequent builds you can simply run
+```
+cd %GOPATH%\src\github.com\ipfs\go-ipfs
+make install
+```
 
-The following commands should download or update go-ipfs dependencies
-and then install them:
+## Minimal
+While it's possible to build `go-ipfs` with `go` alone, we'll be using `git` and `gx` for practical source management.  
+You can use whichever version of `git` you wish but the Windows builds at <https://git-scm.com> are recommended. `git` must be in your `PATH` for `go get` to recognize and use it.  
+`gx` and its go-hook `gx-go` will be built by us. Alternativley you may use [prebuilt binaries](https://dist.ipfs.io/#gx) by placing them in your `PATH` and skipping `go get` for `gx` and `gx-go` below.  
 
 ```
+SET PATH=%PATH%;%GOPATH%\bin
 go get -u github.com/whyrusleeping/gx
 go get -u github.com/whyrusleeping/gx-go
+go get -u -d github.com/ipfs/go-ipfs
 cd %GOPATH%/src/github.com/ipfs/go-ipfs
 gx --verbose install --global
-```
-
-## Manual Way: build go-ipfs
-
-To actually build go-ipfs, first go to the cmd/ipfs directory:
-
-```
 cd cmd\ipfs
 ```
-
-Then get the current Git commit:
-
-```
-git rev-parse --short HEAD
-```
-
-It will output a small number of hex characters that you must pass to
-the actual build command (replace XXXXXXX with these characters):
-
-```
-go install -ldflags="-X "github.com/ipfs/go-ipfs/repo/config".CurrentCommit=XXXXXXX"
-```
-
-After that ipfs should have been built and should be available in
-"%GOPATH%\bin".
-
-You can check that the ipfs you built has the right version using:
-
-```
-ipfs version --commit
-```
-
-It should output something like "ipfs version 0.4.0-dev-XXXXXXX" where
-XXXXXXX is the current commit that you passed to the build command.
-
-## Cygwin way: install Cygwin
-
-Install Cygwin as explained in the Cygwin documentation:
-
-http://cygwin.com/install.html
-
-By default Cygwin will not install 'make', so you should click on the
-"Devel" category during the Cygwin installation process and then check
-the 'make' package.
-
-## Cygwin way: build go-ipfs
-
-To build go-ipfs using Cygwin you just need to open a Cygwin Terminal
-and then type the following commands:
-
-```
-cd $GOPATH/src/github.com/ipfs/go-ipfs
-make install
-```
-
-After that ipfs should have been built and should be available in
-"%GOPATH%\bin".
-
-You can check that the ipfs you built has the right version using:
+The syntax for the next command is different depending on how it's executed. If you're on the interactive command line vs executing from inside the batch/cmd interpriter.  
+interactive: `FOR /F %V IN ('git rev-parse --short HEAD') do set SHA=%V`  
+interpreter: `FOR /F %%V IN ('git rev-parse --short HEAD') do set SHA=%%V`
 
 ```
-ipfs version --commit
+go install -ldflags="-X "github.com/ipfs/go-ipfs/repo/config".CurrentCommit=%SHA%"
```

Comment:
I think it would definitely be helpful to note that this is the actual build step. It could be as simple as just saying “finally, actually build and install `go-ipfs`.”